systray_windows.go

```diff
@@ -489,6 +591,50 @@ func (t *winTray) addOrUpdateMenuItem(item *MenuItem) error {
 	return nil
 }
 
+func (t *winTray) addBitMapMenuItem(bitmapPath string, item *MenuItem) error {
+	const (
+		IMAGE_BITMAP    = 1          // Loads a bitmap
+		LR_LOADFROMFILE = 0x00000010 // Loads the stand-alone image from the file
+	)
+	// Create a handle to the bitmap
+	srcPtr, err := windows.UTF16PtrFromString(bitmapPath)
+	fmt.Println(bitmapPath, srcPtr, err)
+	if err != nil {
+		return err
+	}
+	res, _, err := pLoadImage.Call(
+		0,
+		uintptr(unsafe.Pointer(srcPtr)),
+		IMAGE_BITMAP,
+		16,
+		16,
+		LR_LOADFROMFILE,
+	)
+	fmt.Println("LoadImage: ", res)
+	if res == 0 {
+		return err
+	}
+
+	h := windows.Handle(res)
+	fmt.Println("BitmapHandle: ", h)
+	res, _, err = pGetMenuItemID.Call(uintptr(t.menu), uintptr(item.id))
+	fmt.Println("GetMenuID: ", res)
+	fmt.Println(item.id)
+	res, _, err = pSetMenuItemBitmaps.Call(
+		uintptr(t.menu),
+		uintptr(item.id),
+		0,
+		uintptr(h),
```

Comment:
@amkulikov I am trying to add a Bitmap image to a menuItem but its not working, what am i doing wrong or how can i debug this?
You can build the example i added and check that its not working..

Also i added submenus, (the code needs ironing) for both macOS and windows and it works, also your commits didn't break the mac builds.